<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>148</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>307</width>
         <height>69</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>309</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSave"/>
    <addaction name="separator"/>
    <addaction name="actionFilePrint"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Правка</string>
    </property>
    <addaction name="actionEditCopy"/>
    <addaction name="actionEditPaste"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;Вид</string>
    </property>
    <addaction name="actionViewZoomIn"/>
    <addaction name="actionViewZoomOut"/>
    <addaction name="actionViewSizeNormal"/>
    <addaction name="separator"/>
    <addaction name="actionViewSizeAdjust"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileOpen"/>
   <addaction name="actionFileSave"/>
   <addaction name="separator"/>
   <addaction name="actionFilePrint"/>
   <addaction name="separator"/>
   <addaction name="actionEditCopy"/>
   <addaction name="actionEditPaste"/>
   <addaction name="separator"/>
   <addaction name="actionViewZoomIn"/>
   <addaction name="actionViewZoomOut"/>
   <addaction name="actionViewSizeNormal"/>
   <addaction name="actionViewSizeAdjust"/>
  </widget>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/fileOpen.ico</normaloff>:/pic/fileOpen.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть изображение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/fileSave.ico</normaloff>:/pic/fileSave.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить изображение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFilePrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/filePrint.ico</normaloff>:/pic/filePrint.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Печать</string>
   </property>
   <property name="toolTip">
    <string>Печать изображения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/fileExit.ico</normaloff>:/pic/fileExit.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход из программы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEditCopy">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/editCopy.ico</normaloff>:/pic/editCopy.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Копировать</string>
   </property>
   <property name="toolTip">
    <string>Копировать изображение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionEditPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/editPaste.ico</normaloff>:/pic/editPaste.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Вставить</string>
   </property>
   <property name="toolTip">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionViewZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/zoomIn.ico</normaloff>:/pic/zoomIn.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Увеличить</string>
   </property>
   <property name="toolTip">
    <string>Увеличить изображение</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionViewZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/zoomOut.ico</normaloff>:/pic/zoomOut.ico</iconset>
   </property>
   <property name="text">
    <string>У&amp;меньшить</string>
   </property>
   <property name="toolTip">
    <string>Уменьшить изображение</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/helpAbout.ico</normaloff>:/pic/helpAbout.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;О программе...</string>
   </property>
   <property name="toolTip">
    <string>О программе...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionViewSizeNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/zoomNorm.ico</normaloff>:/pic/zoomNorm.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Исходный</string>
   </property>
   <property name="toolTip">
    <string>Исходный размер изображения</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="actionViewSizeAdjust">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pic/zoomFit.ico</normaloff>:/pic/zoomFit.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Авто</string>
   </property>
   <property name="toolTip">
    <string>Автоматический размер изображения</string>
   </property>
   <property name="shortcut">
    <string>End</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
